commands/initial/templates/webpack/webpack.dev.js

```diff
@@ -29,7 +29,6 @@ module.exports = webpackCommon('dev', {
     devtool: 'cheap-module-eval-source-map',
     entry: {
         app: [ examplePath('example.main') ],
-        scripts: [],
```

Comment:
Oh really? Interesting. Maybe that was an update. We'll keep it out for now...I might have even made that change, I can't remember 😉